Doc/library/asyncio-eventloop.rst

```diff
@@ -188,19 +188,21 @@ Running and stopping the loop
 .. coroutinemethod:: loop.shutdown_default_executor(timeout=None)
 
    Schedule the closure of the default executor and wait for it to join all of
-   the threads in the :class:`ThreadPoolExecutor`. After calling this method, a
-   :exc:`RuntimeError` will be raised if :meth:`loop.run_in_executor` is called
+   the threads in the :class:`ThreadPoolExecutor`. After calling this method,
+   :meth:`loop.run_in_executor` will raise a :exc:`RuntimeError` if called
    while using the default executor.
 
-   The *timeout* parameter specifies the amount of time the executor will
-   be given to finish joining. The default value is ``None``, which means the
-   executor will be given an unlimited amount of time.
+   The *timeout* parameter specifies the amount of time this method gives the
+   executor to finish joining. The default value is ``None``, which means the
```

Comment:
```suggestion
   *timeout* specifies the duration in seconds (as a :class:`float`)
   this method gives the executor to finish joining.
   With the default, ``None``, the
```

* Add key clarifying detail that timeout is in seconds as a `float`, which is never stated, and I had to dig into [the code](https://github.com/python/cpython/blob/5d09d11aa0b89aeba187f4f520728ccaf4fc5ac1/Lib/threading.py#L1094) to be sure. This leaves the reader to guess—Float seconds? Integer microseconds? `datetime` objects? `time` structs? Per Diataxis, there should be no doubt or ambiguity in reference.
* Be more concise, direct and to the point, as appropriate for reference docs